packages/core/src/components/slider/multiSlider.tsx

```diff
@@ -99,7 +99,7 @@ export interface ISliderBaseProps extends IProps, IIntentProps {
      *
      * @default true
      */
-    labelRenderer?: boolean | ((value: number) => string | JSX.Element);
+    labelRenderer?: boolean | ((value: number, isHandle?: boolean) => string | JSX.Element);
```

Comment:
for a bit of forwards compatibility if we add more options here:

```suggestion
    labelRenderer?: boolean | ((value: number, opts?: { isHandleTooltip: boolean; }) => string | JSX.Element);
```